.github/workflows/pull-request-target.yml

```
@@ -0,0 +1,77 @@
+          message: |
+            ## Our Pull Request Approval Process
+            
+            Thanks for contributing! 
+            
+            ### Testing Your Code
+            
+            Remember, your PRs won't be reviewed until these criteria are met:
+            
+            1. We don't merge PRs with poor code quality. 
+                1. Follow coding best practices such that CodeRabbit.ai approves your PR.
+            1. We don't merge PRs with failed tests. 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)